middleware/rewrite_test.go

```diff
@@ -66,14 +66,14 @@ func TestEchoRewritePreMiddleware(t *testing.T) {
 
 	// Route
 	r.Add(http.MethodGet, "/new", func(c echo.Context) error {
-		return c.NoContent(200)
+		return c.NoContent(http.StatusOK)
 	})
 
 	req := httptest.NewRequest(http.MethodGet, "/old", nil)
 	rec := httptest.NewRecorder()
 	e.ServeHTTP(rec, req)
-	assert.Equal(t, "/new", req.URL.EscapedPath())
-	assert.Equal(t, 200, rec.Code)
+	assert.Equal(t, "/new", req.URL.Path)
```

Comment:
Please keep req.URL.EscapedPath() for consistency in the tests (interoduced some commits ago)